recipes/pyhocon/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.3.30" %}
+{% set md5 = "b12316376427cc0e5da5a2fd8ef116e8" %}
+
+package:
+  name: pyhocon
+  version: {{ version }}
+
+source:
+  fn: pyhocon-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/58/b2/8eac351069942d8ba77c664df1368519e1679f5c729d5f5affe4f75a512a/pyhocon-{{ version }}.tar.gz
```

Comment:
For a simpler, more maintainable way to do this using templates, see https://github.com/conda-forge/staged-recipes/pull/1132/files